consensus/src/epoch_manager.rs

```diff
@@ -337,16 +331,16 @@ impl<T: Payload> EpochManager<T> {
             self.storage.clone(),
             self.time_service.clone(),
         );
-        processor.start().await;
-        self.processor = Some(Processor::EventProcessor(processor));
-        info!("EventProcessor started");
+        processor.start(last_vote).await;
+        self.processor = Some(RoundProcessor::Normal(processor));
+        info!("RoundManager started");
```

Comment:
Should we unify the RoundManager vs RoundProcessor naming?

Reply:
I have difficulty figuring out the name for the RoundProcessor, it's a enum with RecoveryManager + RoundManager, suggestions are welcome!